kernel/declarations.ml

```diff
@@ -52,7 +52,7 @@ type inline = int option
 type projection_body = {
   proj_ind : MutInd.t;
   proj_npars : int;
-  proj_arg : int;
+  proj_arg : int; (** Projection index, starting from 0 *)
```

Comment:
I believe this includes parameters, ie `proj_arg` for `fst` is `2`. It may be worth checking and including this information in the comment.

Reply:
Nope, I checked, this is really the projection index.